.gitignore

```diff
@@ -34,6 +34,7 @@ bower_components
 .lock-wscript
 
 # Compiled binary addons (http://nodejs.org/api/addons.html)
+lib/
 build/
 prebuilds/
```

Comment:
You might as well remove this `prebuilds/` which is only required for `prebuildify`, or use the same folder and avoid changing this file at all.